### PR TITLE
feature(build): adds gulp autoprefixer

### LINKED ISSUE #11: Prefer autoprefixer in gulp over mixins
I started with a couple of simple mixins for things that need vendor prefixes, but this is already annoying me. Lose those in favor of a gulp autoprefixer solution.

### PR DESCRIPTION
**Pull request by chriswhawkins**
Adds autoprefixer to the dependencies and the default gulp task

Closes #11

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -12,6 +13,18 @@ gulp.task('sass', function(){
     }))
 });
 
+gulp.task('autoPrefix', function () {
```
Comment (mewdriller) [inline gulpfile.js]: Typically I've seen task names all lowercase with dashes between multiple words and colons between groupings. I probably would have expected either `autoprefix` or `styles:autoprefix` for this one.

Comment (mewdriller) [inline gulpfile.js]: In a similarly nitpicky fashion, the spacing around the `function () {` here looks different from the rest of the file I can see in the diff; e.g.: `function() {`.

Clearly not a big deal, but just something that I noticed.

**Comment by mewdriller** [comment]
LGTM.

I had a few :art:-type comments, but that's all. Looks good @chriswhawkins. :shipit:

**Comment by chriswhawkins** [comment]
Fixed that code formatting. Thanks @mewdriller

**Pull request merged**